### DISCUSSION TITLE
[Feat]: Ability to change easing function on slide change

### DISCUSSION THREAD

**Discussion by Taimoor-Tariq** [davidjerleke/embla-carousel#733] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Right now the easing function is "ease-in" when the next or previous buttons are pressed. Is there a way to change this behaviour

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@Taimoor-Tariq thanks for your question. At the time of writing it's not doable. Please read this:

- https://github.com/davidjerleke/embla-carousel/issues/184#issuecomment-827396864
